frontend/index.html

```diff
@@ -783,7 +856,11 @@ <h2 data-i18n="contact">Contact</h2>
         // https://stackoverflow.com/questions/1414365/disable-enable-an-input-with-jquery
         Promise.all(otherI18nLoadPromises).then(() => {
             console.log("Finished loading other languages "+toLoadLanguages);
-            $('#langSel').prop('disabled', false)
+            //now that all i18n have been loaded, get map
+            LANG_TO_NAME = makeKeyNameMap(SUPPORTED_LANGUAGES);
```

Comment:
This is where I've determined to load the code to name mapping and create the menu, since the i18n information exists here

Reply:
Yup, this is correct.